src/test/java/CI/GitHandlerTest.java

```diff
@@ -6,6 +6,28 @@
 
 public class GitHandlerTest {
 
+  /**
+    * This tests the start_tests function in the GitHandler class.
+    * @author Philippa Ö, Andreas G.
+    */
+
+   @Test
+   public void testStartTests() {
+     // We pull a branch and run tests on this one
+     GitEvent event = new GitEvent("", "");
+     event.setBranchName("test");
```

Comment:
Actually, the dummy branch we created to test successful build and tests is "buildstestspass", not "test".